code/obj/shieldgen.dm

```diff
@@ -156,16 +156,6 @@ Shield and graivty well generators
 
 		return 1
 
-/obj/shieldwall
```

Comment:
You removed shieldwalls, either by accident or on purpose.

Reply:
this is the shorter of two duplicate definitions, the other still in [code/obj/item/device/energy_shield_gen.dm](https://github.com/goonstation/goonstation/pull/16045/files/9fa500f956fc07fc939ce8696d972ae6d5ba1749#diff-457e91bffd5a4a80de7d782f30cfe2ffa995677e7ed406df48f974c069e81919)